test/org/daisy/dotify/formatter/impl/FormatterImplTest.java

```diff
@@ -108,19 +90,284 @@ public void newSection(SectionProperties props) {
             }
 
             @Override
-            public void newRow(Row row) {
-                sb.append(row.getChars());
-            }
+            public void newRow(Row row) { sb.append(row.getChars()); }
```

Comment:
Suggestion: if you also insert a newline (or something else that makes it more obvious that it's a new line) here that could make it easier to read the tests. `newPage` could insert some other character(s) that indicates a page break.